classes/class-install.php

```diff
@@ -71,9 +71,6 @@ public function __construct( $plugin ) {
 		$this->db_version = $this->get_db_version();
 		$this->stream_url = self_admin_url( $this->plugin->admin->admin_parent_page . '&page=' . $this->plugin->admin->settings_page_slug );
 
-		// Check DB and display an admin notice if there are tables missing.
-		add_action( 'init', array( $this, 'verify_db' ) );
-
 		// Install the plugin.
 		add_action( 'wp_stream_before_db_notices', array( $this, 'check' ) );
```

Comment:
I think this can also be removed since we're removing the `wp_stream_before_db_notices` call already.

Reply:
@kidunot89 Can we remove this now that the `do_action( 'wp_stream_before_db_notices' )` is being removed?